gengokit/httptransport/httptransport.go

```diff
@@ -246,6 +259,7 @@ func createDecodeConvertFunc(f Field) string {
 		fType = "%s, err := strconv.ParseFloat(%s, 64)"
 	case strings.Contains(f.GoType, "string"):
```

Comment:
Why is this `strings.Contains` instead of `==`? Is there a specific reason for this wider match that I am missing?